packages/ckb/src/rgbpp/btc-time.ts

```diff
@@ -128,13 +128,15 @@ export const signBtcTimeCellSpentTx = async ({
   ckbFeeRate,
 }: SignBtcTimeCellsTxParams): Promise<CKBComponents.RawTransaction> => {
   const masterLock = addressToScript(masterCkbAddress);
-  const emptyCells = await collector.getCells({
+  let emptyCells = await collector.getCells({
     lock: masterLock,
+    isDataEmpty: true,
     outputCapacityRange,
   });
   if (!emptyCells || emptyCells.length === 0) {
     throw new Error('No empty cell found');
   }
+  emptyCells = emptyCells.filter((cell) => !cell.output.type);
   const emptyInput: CKBComponents.CellInput = {
```

Comment:
code search: https://github.com/search?q=repo%3Ackb-cell%2Frgbpp-sdk+emptyCells+%3D+await+collector.getCells&type=code

other similar code with `emptyCells = await collector.getCells` may need to be updated too?
- isDataEmpty: true
- emptyCells = emptyCells.filter((cell) => !cell.output.type)

Reply:
Under normal circumstances, cell data is not empty, meaning the cell type script is not null and vice versa is also true.

The `isDataEmpty` here defaults to true, so there is no problem keeping the original code in most cases.

I once accidentally created a cell during the debug phase. Its cell data was empty, but its cell type was not null. Even if the type is set to null during filtering, the returned cell list will still include this cell.

So I added a layer of code to filter type script here. To be on the safe side, this filtering should be added elsewhere.

Reply:
I see, `isDataEmpty` is default to true.

https://github.com/ckb-cell/rgbpp-sdk/blob/dbb81e36f61fae9a8eeb64d3fd022d7b051cf056/packages/ckb/src/collector/index.ts#L32